samples/stored-procedures/length.js

```diff
@@ -0,0 +1,90 @@
+/**

```

Comment:
Can this be renamed to count_underscoreAPI.js to make it clear this is similar to bulkDelete.js but using underscore API?